app/create_file.py

```diff
@@ -1 +1,48 @@
-# write your code here
+import os
+import sys
+from datetime import datetime
+
+
+def get_clean_command(cmd: str) -> str:
+    return " ".join(sys.argv).split(cmd)[1]
+
+
+def file_creation(file_path: str) -> None:
+    with open(file_path, "a") as file:
+        line = 1
+        file.write(
+            datetime.now().strftime("%Y-%m-%d %H:%M:%S") + "\n"
+        )
+        while True:
+            reply = input("Enter content line: ")
+            if reply == "stop":
+                file.write("\n")
+                break
+            file.write(f"{line} {reply}\n")
+            line += 1
+
+
+def path_operations() -> None:
+    file_path, new_file = None, None
+
+    if "-d" in sys.argv:
+        command_dirs = get_clean_command("-d")
+        if "-f" in command_dirs:
+            command_dirs, new_file = command_dirs.split("-f")
+
+        path = os.path.join(*command_dirs.split())
+        os.makedirs(path, exist_ok=True)
+
+        if new_file:
+            file_path = os.path.join(path, new_file)
+    elif "-f" in sys.argv:
+        file_path = get_clean_command("-f")
+    else:
+        raise ValueError("No '-f' or '-d' command was passed!")
+
+    if file_path:
+        file_creation(file_path)
```

Comment:
else is excessive, you could handle it this way, or similar
```suggestion
    if not file path:
        raise ValueError("No '-f' or '-d' command was passed!")

    file_creation(file_path)
```

Reply:
I don't think so, because if-elif-else statement here checks what exactly to create - a directory path or a file one. So basically that `raise` statement tells when no command was passed. 
If I move it to the `if file_path` block then the user will get an error when he/she creates just a directory path.

Hope you got my idea)